src/views/CampusSafety/views/LostAndFound/views/MissingItemEdit/index.tsx

```diff
@@ -208,6 +230,41 @@ const MissingItemFormEdit = () => {
             <GordonLoader />
           ) : (
             <>
+              {/* Display the "Found" notice only if the status is "found" */}
+              {formData.status.toLowerCase() === 'found' && (
+                <Box className={styles['found-container']}>
+                  {/* Status Message */}
+                  <Box className={styles['status-message']}>
```

Comment:
Why ```styles['status-message']``` and not ```styles.status-message``` like elsewhere in 360 and in our project?

Reply:
i can change that, just was testing if it would work that way, and didnt change it back

Reply:
Yeah, fair, I'm just thinking for consistency it'd be good to keep them the same as others, not a huge deal, but should be a simple change.

Reply:
Resolved this.